src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -29,24 +31,17 @@
     /**
      * Convenience constructor using raw values.
      *
-     * @throws IllegalValueException if any of the raw values are invalid
+     * @throws IllegalValueException
+     *             if any of the raw values are invalid
      */
-    public AddCommand(String name,
-                      String phone, boolean isPhonePrivate,
-                      String email, boolean isEmailPrivate,
-                      String address, boolean isAddressPrivate,
-                      Set<String> tags) throws IllegalValueException {
+    public AddCommand(String name, String phone, boolean isPhonePrivate, String email, boolean isEmailPrivate,
+            String address, boolean isAddressPrivate, Set<String> tags) throws IllegalValueException {
```

Comment:
Which indentation is more readable in this case?
Would the former indentation style be more readable than the latter implementation?